lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -79,8 +79,27 @@ def self.create_milestone(repository, newmilestone_number, newmilestone_duedate,
         github_client().create_milestone(repository, newmilestone_number, options)
       end
 
-      def self.create_release(repository, version, release_notes, assets, prerelease)
-        release = github_client().create_release(repository, version, name: version, draft: true, prerelease: prerelease, body: release_notes)
+      # Creates a Release on GitHub as a Draft
+      #
+      # @param [String] repository The repository to create the GitHub release on. Typically a repo slug (<org>/<repo>).
+      # @param [String] version The version for which to create this release. Will be used both as the name of the tag and the name of the release.
+      # @param [String?] target The commit sha1 or branch name that this release will point to when it's published and creates the tag.
+      #        If nil (the default), will use the repo's current HEAD commit at the time this method is called.
+      #        Unused if the tag already exists.
+      # @param [String] body The text to use as the release's body / description (typically the release notes)
+      # @param [Array<String>] assets List of file paths to attach as assets to the release
+      # @param [TrueClass|FalseClass] prerelease Indicates if this should be created as a pre-release (i.e. for alpha/beta)
+      #
+      def self.create_release(repository:, version:, target: nil, body:, assets:, prerelease:)
+        release = github_client().create_release(
+          repository,
+          version, # tag name
+          name: version, # release name
+          target_commitish: target || Fastlane::Helper::GitHelper.get_commit_sha,
```

Comment:
TIL "commitish":

![image](https://user-images.githubusercontent.com/1218433/118066941-ecb1fb00-b3e2-11eb-9b0d-075552309463.png)

Reply:
Is there a use case for us to support the commitish format? That is, do we need support for creating a release on the tip of a branch? As far as I can see, I don't think we do.

Would removing the support for the branch simplify things? If not the coding itself, it may make the naming clearer, from `target` to `sha1` for example.

Reply:
Yeah I used that name because both options were supported out of the box anyway. I mean that parameter is used verbatim without any transformation and directly passed to the call to the GitHub API, and that API support both. So I didn't see a reason why we'd forcefully limit that parameter to only be a SHA1. Besides, how would you do that sort of limitation anyway and why?

One thing we could consider doing is to not have the action param / ConfigItem at all, and always use the current SHA without any way to override it. After all that's currently the only use case we have so far. But I thought it was nice and cheap to expose to option to override for the future (I know I know YAGNI 😅)